libs/coin-modules/coin-cardano/src/js-signOperation.ts

```diff
@@ -206,6 +188,56 @@ const buildOptimisticOperation = (
   return op;
 };
 
+/**
+ * Sign Transaction with Ledger hardware
+ */
+const buildSignOperation =
+  (signerContext: SignerContext<CardanoSigner>): SignOperationFnSignature<Transaction> =>
```

Comment:
Just a note but we could also type `SignOperationFnSignature<CardanoAccount, Transaction>`